libs/datamodel/core/tests/types/mongo_native_types.rs

```diff
@@ -1437,6 +1311,13 @@ fn invalid_json_usage_in_model() {
         [1;94m16 | [0m  d  Boolean  @test.Json
         [1;94m17 | [0m  e  DateTime @[1;91mtest.Json[0m
         [1;94m   | [0m
+        [1;91merror[0m: [1mError validating field `f` in model `A`: Field `f` in model `A` can't be of type Decimal. The current connector does not support the Decimal type.[0m
+          [1;94m-->[0m  [4mschema.prisma:18[0m
+        [1;94m   | [0m
+        [1;94m17 | [0m  e  DateTime @test.Json
+        [1;94m18 | [0m  [1;91mf  Decimal  @test.Json[0m
+        [1;94m19 | [0m}
+        [1;94m   | [0m
```

Comment:
Is this one expected?

Reply:
Originally I thought that I don't want to remove the test and this error is fine, but failed to look at the other errors above, which are not great. It's probably better to remove the entire native type for now.